bin/env_variables_all.txt

```diff
@@ -108,3 +108,12 @@ TEST_URL=http://localhost:4452
 USER=someuser
 USER_PROFILE_HOST=http://ccd-user-profile-api:4453
 USER_PROFILE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_definition,ccd_admin
+ROLE_ASSIGNMENT_USER_EMAIL=ccd.ac.solicitor1@gmail.com
+ROLE_ASSIGNMENT_USER_PASSWORD=Pa55word11
+ROLE_ASSIGNMENT_HOST=http://localhost:4096
+ROLE_ASSIGNMENT_API_GATEWAY_S2S_CLIENT_ID=ccd_data
+ROLE_ASSIGNMENT_API_GATEWAY_S2S_CLIENT_KEY=AAAAAAAAAAAAAAAB
+ROLE_ASSIGNMENT_SUPER_USER=ccd.ac.superuser@gmail.com
+ROLE_ASSIGNMENT_SOLICITOR_USER=ccd.ac.solicitor1@gmail.com
+ROLE_ASSIGNMENT_SOLICITOR_USER_PWD=Pa55word11
+ROLE_ASSIGNMENT_SUPER_USER_PWD=Pa55word11
```

Comment:
Just noticed befta pr also references ROLE_ASSIGNMENT_STAFF1_USER, ROLE_ASSIGNMENT_STAFF1_USER_PWD, ROLE_ASSIGNMENT_STAFF2_USER and ROLE_ASSIGNMENT_STAFF2_USER_PWD can we add them here?

Reply:
Added the details here and ccd-aat as well.